airbyte-integrations/connectors/source-github/source_github/schemas/deployments.json

```diff
@@ -0,0 +1,121 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
```

Comment:
` ["null", "object"] ` is better, to avoid empty object processing by `json_schema_helper`